next.config.ts

```diff
@@ -5,24 +5,34 @@ const nextConfig: NextConfig = withPlausibleProxy()({
   async headers() {
     return [
       {
-        source: '/(.*)',
+        source: "/(.*)",
```

Comment:
this is just `prettier` running on the file, IDK if we have an opinion in this repo